target/linux/airoha/files/drivers/pinctrl/pinctrl-airoha.c

```diff
@@ -1318,12 +1436,107 @@ static const struct airoha_pinctrl_func_group pwm_func_group[] = {
 		.mux_func = AIROHA_FUNC_PWM_EXT_MUX,
 		.regs[0] = {
 			REG_GPIO_FLASH_MODE_CFG_EXT,
+			GPIO47_FLASH_MODE_CFG,
 			GPIO47_FLASH_MODE_CFG
 		},
 		.num_regs = 1,
 	},
 };
 
+static const struct airoha_pinctrl_func_group serdes_func_group[] = {
+	{
+		.name = "usb_hsgmii",
+		.mux_func = AIROHA_FUNC_SERDES_SSR3,
+		.regs[0] = {
+			REG_SSR3,
+			SSUSB_HSGMII_SEL,
+			SSUSB_HSGMII_SEL
+		},
+		.num_regs = 1,
+	},
```

Comment:
@merbanan no new-line here (and for all other cases below)

Reply:
Ack